### ISSUE TITLE
indication of reliability for dates

### ISSUE THREAD

**Issue by eroux** [BuddhistDigitalResourceCenter/owl-schema#91]
There's no need to rush on that as I think a more global solution will need to be crafted, but it's worth reporting that we cannot yet fully record the ToL information: they have a boolean expressing the fact that a birth or death date is approximate or not, and there's currently no obvious way to express that in our vocabulary. We need to discuss at some point how we'll record that.

**Comment by xristy** [comment]
that was being done in the legacy system via the use of "u" and "?" following a subset of a LoC date convention. It would obviously be better with appropriate properties rather than buried in the date data. There needs to be some way indicating a decade or century and to indicate that there is uncertainty regarding the date specification. Hence "182u" means "sometime between 1820 and 1829" while "182u?" means "sometime between 1820 and 1829 but this interval is itself uncertain". Also needed is a more general interval such as the Birth of P314: 1729/1730, which could be 1729-30.

**Comment by eroux** [comment]
I think the `:notBefore` and `:notAfter` can be used for the interval, maybe we could just have a new property to indicate uncertainty? something like `:dateIsApproximative` with a boolean?

**Comment by xristy** [comment]
I agree that the ``:notBefore`` and ``:notAfter`` can be used for intervals. I'll add ``:dateApproximate``.

I noticed that in some cases such as ``bdr:W20006`` there needs to be some adjustments during xmltold... The legacy data has _Publisher Date  2000-2_ which migrates as:

    [] :notBefore 2000 ;
       :notAfter 2 .
